apps/emqx_exhook/test/emqx_exhook_demo_svr.erl

```diff
@@ -80,7 +80,10 @@ stop() ->
 
 stop(Name) ->
     grpc:stop_server(Name),
-    to_atom_name(Name) ! stop.
+    case whereis(to_atom_name(Name)) of
+        undefined -> ok;
+        Pid -> Pid ! stop
```

Comment:
nit: for deterministic, monitor this pid, and wait for the DOWN message.

Reply:
Thanks for your advices :]